source/_assets/sass/main.scss

```
@@ -1,9 +1,7 @@
 @tailwind base;
 @tailwind components;
 
-// Code syntax highlighting,
-// powered by https://highlightjs.org
-@import '~highlight.js/styles/nnfx-dark.css';
+@import 'highlight.js/styles/nnfx-dark.css';
```

Stylelint (17.12.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)